### DISCUSSION TITLE
How can I hide or move the keyboard shortcut hints?

### DISCUSSION THREAD

**Discussion by rpbaptist** [ibhagwan/fzf-lua#1287] [Q&A] [answered]
I'd like the search result to be as close to my cursor as possible. Instead I'm seeing these hints:

![image](https://github.com/ibhagwan/fzf-lua/assets/7916262/0732ae0f-9070-42a4-ab91-50281dcf1d98)

How can I move this text to the bottom, to the right window or just hide it?

**Comment by ibhagwan** [comment, marked as answer]
See #1133

**Reply by rpbaptist** [reply]
Thank you.
